obspy/core/inventory/inventory.py

```diff
@@ -911,8 +912,8 @@ def _get_epoch_plottable_struct(self):
             else:
                 end = min(self.end_time, UTCDateTime.now())
```

Comment:
I think we should plot what really is given for the endtime of Inventories etc. and not cut them off at current time. I'm assuming you did this to get the right end of the plot at current time? This can also be achieved by just setting the right x limit to current time at the very end of all plotting before popping up the plot window. But when people zoom out interactively, the actual end time can be seen.

For open ended epochs we could just leave away the end marker and have them end at current time